### PR TITLE
update transcript for 4_2

### LINKED ISSUE #47: Improve episode transcription Season 4 Episode 2
## Issue Context

Our podcast's transcriptions are automatically generated, so there would be typos or missing words. 

We want to improve every episode's transcription so they can be accessible to everyone. Currently, we are doing this manually.

## Steps To Update

- Open the file of the podcast's episode. 
  e.g.: `1_0.srt` means season 1 episode 0
-  Listen to the podcast's episode and improve the transcript based on your hearing.
-  [Fix the format](https://github.com/Virtual-Coffee/podcast-transcripts#fixing-formatting-issues) by running `yarn srt-check`.
- After running `yarn srt-check`, fix timestamps suggestions format manually, if any.

## Accessibility Resources

- [Transcribing Audio to Text - W3C WAI](https://www.w3.org/WAI/media/av/transcribing/)

If you have questions or need help, please let us know.

### PR DESCRIPTION
**Pull request by ClJarvis**
### Description
Improved auto generated transcripts for  Season 4 Episode 2 with Todd Libby.
Ran yarn check-srt to fix incorrect timestamp formatting.
Fixed colon error

closes #47

### CODE REVIEW

**Comment by danieltott** [comment]
@adiati98 can you take a look at this when you get a chance?

**Comment by adiati98** [comment]
@danieltott sure!

File: episodes/4_2.srt [hunk 1]
```diff
@@ -1,4180 +1,4264 @@
...
+93
 00:04:22,803 --> 00:04:24,394
 emoji pop up. You're like, oh,
 cool. Todd's here.
 
-89
+94
 00:04:26,093 --> 00:04:28,255
-Bekah: I just think it's it's
+Bekah: I just think it's-it's
```
Comment (adiati98) [inline episodes/4_2.srt:437]: We want to give a space after the dash for repeated word.

e.g.:

```markdown
Bekah: I just think it's- it's
```

File: episodes/4_2.srt [hunk 2]
```diff
@@ -1,4180 +1,4264 @@
...
-anybody remembers those, choose
+anybody remembers those, "Choose
 
-131
-00:06:49,564 --> 00:06:52,009
-your own adventure books from
+137
+00:06:49,564 --> 00:06:50,786
+Your Own Adventure" books from
 way back in the day. I, I used
```
Comment (adiati98) [inline episodes/4_2.srt:638]: I thinks this one is a repeat word?

`way back in the day. I- I used`

File: episodes/4_2.srt [hunk 3]
```diff
@@ -1,4180 +1,4264 @@
...
-279
+291
 00:16:18,216 --> 00:16:21,471
 scans
 
-280
+292
 00:16:21,471 --> 00:16:27,772
 first. So I use stuff like the
 WAVE extension, um, the axe, I
```
Comment (adiati98) [inline episodes/4_2.srt:1365]: Is `the axe` a name? If so, we need to capitalize here.

File: episodes/4_2.srt [hunk 4]
```diff
@@ -1,4180 +1,4264 @@
...
+309
 00:17:52,701 --> 00:17:58,192
 there's Voiceover. And it's best
 to use Voiceover in Safari
 
-297
+310
 00:17:58,221 --> 00:18:01,686
 because the native, you know,
 Mac app and everything, um, and
```
Comment (adiati98) [inline episodes/4_2.srt:1453]: I believe this is `NVDA` and not `and VDA`

File: episodes/4_2.srt [hunk 5]
```diff
@@ -1,4180 +1,4264 @@
...
-305
+319
 00:18:31,041 --> 00:18:34,236
 are
 
-306
+320
 00:18:34,237 --> 00:18:35,317
-the tools I use.
+the tools I use. ha ha.
```
Comment (adiati98) [inline episodes/4_2.srt:1497]: Capitalize the `Ha ha` after a period.

File: episodes/4_2.srt [hunk 6]
```diff
@@ -1,4180 +1,4264 @@
...
 always true, but when I think of
 it as something, oh, something's
 
-322
-00:19:13,537 --> 00:19:16,807
+336
+00:19:13,537 --> 00:19:15,172
 done, now we do an audit on it.
-You know what I mean? but I was
+You know what I mean? Todd: umm
```
Comment (adiati98) [inline episodes/4_2.srt:1574]: We cannot have colon here.

File: episodes/4_2.srt [hunk 7]
```diff
@@ -1,4180 +1,4264 @@
...
 00:24:32,636 --> 00:24:39,086
 Sentence, I guess that's fine.
 Um, and of course you have some
 
-421
+438
 00:24:39,176 --> 00:24:43,287
 images decorative that, ya know,
-it needs an empty all text. And
+it needs an empty alt text. And
```
Comment (adiati98) [inline episodes/4_2.srt:2050]: Let's keep this uniform with `ALT text` :)

File: episodes/4_2.srt [hunk 8]
```diff
@@ -1,4180 +1,4264 @@
...
+490
 00:27:50,826 --> 00:27:52,506
 and then it'll just read like,
 blah, blah, blah dot JPEG.
 
-473
+491
 00:27:52,506 --> 00:27:54,186
-Right.
+Todd: Yes Dan: Right.
```
Comment (adiati98) [inline episodes/4_2.srt:2299]: We cannot have colon here for the change of speaker.

File: episodes/4_2.srt [hunk 9]
```diff
@@ -1,4180 +1,4264 @@
...
 00:34:54,913 --> 00:35:00,943
 courses on, I guess Udemy and
 platforms like that as well.
 
-589
+608
 00:35:03,853 --> 00:35:06,613
-Bekah: So, this has kind of been
-your focus the entire time
+Bekah: So ki, this has kind of
```
Comment (adiati98) [inline episodes/4_2.srt:2850]: Is this an unfinished word? If so, it should be `So ki- this ...`

File: episodes/4_2.srt [hunk 10]
```diff
@@ -1,4180 +1,4264 @@
...
+704
 00:41:43,891 --> 00:41:46,922
 those websites, if you remember
 those websites
 
-686
+705
 00:41:46,942 --> 00:41:47,612
-Dan: here. Yeah.
+Dan: oh. Yeah.
```
Comment (adiati98) [inline episodes/4_2.srt:3316]: Please capitalize the `Oh`.

File: episodes/4_2.srt [hunk 11]
```diff
@@ -1,4180 +1,4264 @@
...
+840
 00:49:08,713 --> 00:49:10,753
 a parent. I don't know if you're
 a parent, Dan, but.
 
-820
+841
 00:49:10,853 --> 00:49:11,253
-Dan: yeah.
+Dan: hell yeah.
```
Comment (adiati98) [inline episodes/4_2.srt:3956]: Please capitalize `Hell yeah.`

File: episodes/4_2.srt [hunk 12]
```diff
@@ -1,4180 +1,4264 @@
...
-856
+877
 00:51:08,648 --> 00:51:10,989
-know, absolutely ridiculous,
+know, absolutely ridiculous.
 
-857
+878
 00:51:11,494 --> 00:51:11,793
 Todd: yeah.
```
Comment (adiati98) [inline episodes/4_2.srt:4129]: Please capitalize the `Yeah.`

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @ClJarvis,
I've dropped some reviews 😊

Also, we are keeping filler words *except* `um` and `uh`.
So please remove these filler words.

I might missed things, so please re-check and make sure:
- No colon in the middle of the line to prevent problematic behavior on the website.
    When a there's a change of speaker, put them in new line with new timestamp (I gave an example in one of the review).
- Capitalize names, new sentence, etc.
- Use `single dash` instead of `comma` for repeated word(s) and give space aftert the dash.

You can refer to our [guildeline](https://github.com/Virtual-Coffee/podcast-transcripts/blob/main/Transcriptions-Guidelines.md#virtual-coffee-podcast-transcriptions-guidelines) for transcribing.

If you have any question, please let me know.
Thank you, Chris!

File: episodes/4_2.srt [hunk 13]
```diff
@@ -1,4180 +1,4264 @@
 1
-00:00:05,203 --> 00:00:09,223
-Bekah: Hello, and welcome to
-season four, episode two of the
+00:00:05,203 --> 00:00:07,213
+Bekah Hawrot Weigel: Hello, and
+welcome to season four, episode
```
[the remark this answers is not in the document]
Comment (ClJarvis) [reply episodes/4_2.srt:4]: √

File: episodes/4_2.srt [hunk 14]
```diff
@@ -1,4180 +1,4264 @@
...
 taught not too long ago. So, um,
 
-295
-00:17:46,551 --> 00:17:52,701
-and uh, screen readers, I will
-use screen readers. So for Mac
+307
+00:17:46,551 --> 00:17:49,626
+and uh, screen readers, Dan: I
+like that Todd:  I will use
```
[the remark this answers is not in the document]
Comment (ClJarvis) [reply episodes/4_2.srt:1439]: Yes, I did that in a few places. I ran check program and it showed some of them with a warning. I fixed the ones it showed.

**Comment by danieltott** [comment]
Thank you @ClJarvis @adiati98 for the work so far!

@ClJarvis I wanted to call out this comment: https://github.com/Virtual-Coffee/podcast-transcripts/pull/78#discussion_r991322100

Any time there's a speaker change, it'll require a new line/timestamp etc.

File: episodes/4_2.srt [hunk 15]
```diff
@@ -1,4180 +1,4264 @@
...
+280
 00:15:28,466 --> 00:15:32,677
 team as an accessibility
 engineer. Uh, so I look forward
 
-269
+281
 00:15:32,677 --> 00:15:34,366
-to that. Yeah.
+to that. Yeah. Dan: honestly
```
[the remark this answers is not in the document]
Comment (ClJarvis) [reply episodes/4_2.srt:1312]: I'm watching on the site. Is there way to get the more detailed time stamp?

Or do I just estimate and make sure one ends before the next one starts?

File: episodes/4_2.srt [hunk 16]
```diff
@@ -1,4180 +1,4264 @@
...
 Dan: Yeah. Well it's well, sure.
 There's there's that too. Yeah.
 
-60
-00:02:58,683 --> 00:03:00,443
+63
+00:02:58,683 --> 00:02:59,563
 Any Raphael really? I'll take,
-uh, take any version. Sounds
+uh, take any version. Bekah:
```
[the remark this answers is not in the document]
Comment (ClJarvis) [reply episodes/4_2.srt:295]: And that includes a new Number for the time stamp? and those that follow.  New speaker means up the timestamp item number up for all timestamps that follow?

282
00:15:34,100 --> 00:15:34,366
Dan: Honestly

**Comment by ClJarvis** [comment]
I made the changes, fixed capitalization, gave new lines for new speakers, and reran the check.

**Comment by adiati98** [comment]
@ClJarvis Awesome! Thanks!
I will do the last edit from my side. Please give me a day or two (max. 3) for each episode because I'm doing something else too atm 😊

@danieltott I've created remote repo based on @ClJarvis's in my local for last editing.
I might need your help later on how to co-author and push the changes so I don't mess with Chris's PR for Hacktoberfest 😁
Thank you!

**Comment by danieltott** [comment]
@ClJarvis We really prefer each PR to contain one issue's worth of work. If you could revert the changes to `3_9` and add them in a new branch and create a new PR, that would be awesome 👍🏼

**Comment by ClJarvis** [comment]
All ready working on it. Didn't realized that the new episode was pushed to the first PR.

**Comment by ClJarvis** [comment]
@danieltott Both branches should be fixed now.

**Pull request merged**